src/network.zig

```diff
@@ -861,6 +861,17 @@ pub const Protocols = struct {
 			conn.sendUnimportant(id, &noData);
 		}
 	};
+	pub const kick = struct {
```

Comment:
I would prefer to have this as part of the disconnect protocol, you could for example just transfer a one-byte enum with the disconnect reason.